Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/StubIrBridgeBuilder.kt

```diff
@@ -116,50 +106,10 @@ class StubIrBridgeBuilder(
                     ?: return
             val cCallAnnotation = function.annotations.firstIsInstanceOrNull<AnnotationStub.CCall.Symbol>()
                     ?: return
-            val cCallSymbolName = cCallAnnotation.symbolName
-            val (wrapperName, wrapperLines) = generateCCalleeWrapper(origin)
-            simpleBridgeGenerator.insertNativeBridge(
-                    function,
-                    emptyList(),
-                    listOf(
-                        *wrapperLines.toTypedArray(),
-                        "const void* $cCallSymbolName __asm(${cCallSymbolName.quoteAsKotlinLiteral()});",
-                        "const void* $cCallSymbolName = &$wrapperName;"
-                    )
-            )
+            val (wrapperLines) = wrapperGenerator.generateCCalleeWrapper(origin.function, cCallAnnotation.symbolName)
```

Comment:
Why `(wrapperLines)` with braces?

Reply:
Because it's deconstruction of data class.

Reply:
Maybe use explicit field getter instead?

Reply:
Maybe use explicit field getter instead?